src/requirements/Github/components/GithubCommitCountRelative.tsx

```diff
@@ -0,0 +1,97 @@
+import {
+  FormControl,
+  FormErrorMessage,
+  FormLabel,
+  NumberDecrementStepper,
+  NumberIncrementStepper,
+  NumberInput,
+  NumberInputField,
+  NumberInputStepper,
+} from "@chakra-ui/react"
+import { ControlledRelativeTimeInput } from "components/common/RelativeTimeInput"
+import { useController, useFormState } from "react-hook-form"
+import { RequirementFormProps } from "requirements"
+import parseFromObject from "utils/parseFromObject"
+
+const GithubCommitCountRelative = ({ baseFieldPath }: RequirementFormProps) => {
+  const { errors } = useFormState()
+
+  const {
+    field: { ref, value, onChange, onBlur },
+  } = useController({
+    name: `${baseFieldPath}.data.id`,
+    rules: {
+      required: "This field is required.",
+      min: {
+        value: 1,
+        message: "Amount must be greater than 0",
+      },
+    },
+  })
+
+  return (
+    <>
+      <FormControl
```

Comment:
The same applies here, you could reuse the `GithubAccountAgeRelative` component. (Or maybe extract the content of it into a `RelativeFromToFormControls` component, and use that component in both places (inside this component, and the `GithubAccountAgeRelative` component).